src/pages/publication/DescriptionPanel.tsx

```diff
@@ -119,17 +125,22 @@ const DescriptionPanel: FC<DescriptionPanelProps> = ({ goToNextTab, savePublicat
                 )}
               </Field>
             </StyledFieldWrapper>
-            <StyledFieldWrapper>
-              {/* TODO: Use <Chip /> or similar to visualize tags  */}
-              <Field
-                aria-label="tags"
-                name={DescriptionFieldNames.TAGS}
-                label={t('description.tags')}
-                component={TextField}
-                fullWidth
-                variant="outlined"
-              />
-            </StyledFieldWrapper>
+            <StyledTagsField>
+              <FieldArray name={DescriptionFieldNames.TAGS}>
+                {({ name, push, remove }) => (
+                  <ChipInput
+                    value={getObjectValueByFieldName(values, name)}
+                    onAdd={tag => push(tag)}
+                    onDelete={(_, index) => remove(index)}
+                    aria-label="tags"
+                    label={t('description.tags')}
+                    helperText={t('description.tags_helper')}
+                    variant="outlined"
+                    fullWidth
+                  />
+                )}
+              </FieldArray>
+            </StyledTagsField>
```

Comment:
var det sånn at dette påvirket localStorage-skjemaet? burde versjonen bumpes isåfall?

Reply:
skal ha vært array før også